style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  margin: 0;
+}
+
+header {
+  border: 3px solid black;
+  padding: 10px 20px;
+  align-self: stretch;
+}
+
+h2 {
+  align-self: center;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+
+  /* gap: 28px; */
```

Comment:
- [ ] As already pointed out previously, please make sure you don't have any commented out code